apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -1,28 +1,31 @@
-import { LoadingOverlay, Modal, useMantineTheme } from '@mantine/core';
+import { Center, LoadingOverlay, Modal, UnstyledButton, useMantineTheme } from '@mantine/core';
```

Comment:
Maybe we could create our own wrapper for UnstyledButton. I could have taken advantage of it rather than erasing styles using the Button for some of the work I did.

Reply:
Yes, that's a good idea :)